v3/cypress/e2e/smoke/single-codap-smoke.ts

```diff
@@ -0,0 +1,193 @@
+        cy.get('[data-testid="no-cases-message"]').should('exist').and('contain.text', 'no cases')
+  })
+  it("verify Four Seals opens from Hamburger menu and tests graph date and table functionality", () => {
+    // Open Four Seals
+    cy.log("Open Four Seals from Hamburger menu")
+      // hamburger menu is hidden initially
+      cfm.getHamburgerMenuButton().should("exist")
+      cfm.getHamburgerMenu().should("not.exist")
+      // hamburger menu is shows when button is clicked
+      cfm.getHamburgerMenuButton().click()
+      cfm.getHamburgerMenu().should("exist")
+      // clicking Open... item closes menu and shows Open dialog
+      cfm.getHamburgerMenu().contains("li", "Open...").click()
+      cfm.getHamburgerMenu().should("not.exist")
+      cfm.getModalDialog().contains(".modal-dialog-title", "Open")
+      // Example Documents should be selected by default
+      cfm.getModalDialog().contains(".tab-selected", "Example Documents")
+      cfm.getModalDialog().contains(".filelist div.selectable", "Mammals").should("exist")
+      cfm.getModalDialog().contains(".filelist div.selectable", "Four Seals").should("exist")
+      // Selecting Mammals document should load the mammals example document
+      cfm.getModalDialog().contains(".filelist div.selectable", "Four Seals").click()
+      cfm.getModalDialog().contains(".buttons button", "Open").click()
+      cy.wait(1000)
+      // once loaded, Open dialog should be hidden and document content should be shown
+      cfm.getModalDialog().should("not.exist")
+      cy.get(".codap-component.codap-case-table").contains(".title-bar", "Tracks/Measurements").should("exist")
+
+    cy.log("Test date display exists in table")
+      cy.get('button[data-testid="codap-attribute-button date"]').should('exist')
+      table.getGridCell(2, 3, 2).should('be.visible').and("contain", "5/23/2005")
+
+    cy.log("Test date display in bottom axis of graph")
+      cy.dragAttributeToTarget("table", "date", "bottom")
+      cy.get('[data-testid="axis-legend-attribute-button-bottom"]').eq(0).should("have.text", "date")
+      // Check that the date axis contains the year '2005'
+      cy.get('[data-testid="axis-bottom"]')
+      .find('text')
+      .contains('2005')
+      .should('exist')
+
+      // Check the number of tick marks on axis (e.g., ensuring there are 9 months: May to Jan)
+      cy.get('[data-testid="axis-bottom"]')
+      .find('text')
+      .should('have.length', 10) // Adjust this if the expected number changes (currently 9 + ghost div=10)
+
+    cy.log("checks map component")
+      c.getComponentTitle("map").should("have.text", "Measurements")
+      // Some day add a check for map with points, see PT #187534790
+      // Also some day add a legend to the map.
+
+  })
+  // NOTE: This test was drafted on 10/16/2024 but will be skipped since we decided we don't need to
+  // smoke test empty CODAP documents
+  it.skip("verify an empty CODAP document appears", () => {
```

Comment:
I think it's fine to include (i.e. un-skip) this as it's really a test that each individual component can be created from the tool shelf rather than a test that an empty document can be loaded.